packages/core/content-releases/admin/src/components/CMReleasesContainer.tsx

```diff
@@ -252,14 +253,21 @@ const AddActionToReleaseModal = ({
 export const CMReleasesContainer = () => {
   const [isModalOpen, setIsModalOpen] = React.useState(false);
   const { formatMessage, formatDate, formatTime } = useIntl();
-  const { id, slug } = useParams<{
+  const { id, slug, collectionType } = useParams<{
     id: string;
     origin: string;
     slug: string;
     collectionType: string;
   }>();
   const isCreatingEntry = id === 'create';
 
+  const { schema } = unstable_useDocument({
+    collectionType: collectionType!,
+    model: slug!,
+  });
```

Comment:
@joshuaellis is this the best way to get the schema of a content type here?

Reply:
minor feedback about useDocument, I saw that the `id` returned is `create` when the document is being created, while this shouldn't be an issue, users customizing strapi could theoretically set the `documentId` (even if we discourage it) value to other strings than uids.

That means a user could set the document id to 'create', breaking the content manager.

Reply:
The issue i forsee is that we have a `create` route and we look at that param for the id, in the CM we ignore if it's create so it won't break anything but they won't be able to get their documents etc lol otherwise we get a 404 and all the UI around that kicks in 😦

Reply:
I am certainly sure this will not be a problem for people, I dont see why someone should set a document id to be create, and if it is they will realise quickly. 🤞